lib/inspec/resources/package.rb

```diff
@@ -217,7 +219,7 @@ def info(package_name)
       }
     rescue JSON::ParserError => e
       raise Inspec::Exceptions::ResourceFailed,
-            "Failed to parse JSON from `brew` command. Error: #{e}"
+              "Failed to parse JSON from `brew` command. Error: #{e}"
```

Comment:
Please don't reformat on functional PRs.

Reply:
Weird, shouldn't have messed with that :thinking: . Anyway, removed change and pushed again.